tests/lax_numpy_test.py

```diff
@@ -2393,6 +2393,9 @@ def testBroadcastToIntIssue1548(self):
     self.assertAllClose(lnp.broadcast_to(1, (3, 2)), onp.ones((3, 2)),
                         check_dtypes=False)
 
+  def testBroadcastToOnScalar(self):
+    self.assertTrue(isinstance(lnp.broadcast_to(10.0, ()), lnp.ndarray))
```

Comment:
`self.assertIsInstance`?

Reply:
WDYT about also adding a line like

```python
self.assertIsInstance(onp.broadcast_to(10.0, ()), onp.ndarray))
```

in the spirit of self-documenting code?

Reply:
Done.